tools/pubsys/src/aws/ami/mod.rs

```diff
@@ -30,49 +31,47 @@ use serde::{Deserialize, Serialize};
 use snafu::{ensure, OptionExt, ResultExt};
 use std::collections::{HashMap, HashSet};
 use std::path::PathBuf;
-use structopt::{clap, StructOpt};
 use wait::wait_for_ami;
 
 const WARN_SEPARATOR: &str = "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!";
 
 /// Builds Bottlerocket AMIs using latest build artifacts
-#[derive(Debug, StructOpt)]
-#[structopt(setting = clap::AppSettings::DeriveDisplayOrder)]
+#[derive(Debug, Parser)]
 pub(crate) struct AmiArgs {
     /// Path to the image containing the os volume
-    #[structopt(short = "o", long, parse(from_os_str))]
```

Comment:
For some reason I thought we had to do something more for the OsStr, but maybe I was overthinking.